rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -177,15 +179,15 @@ public void build_pegout_tx_from_erp_federation() {
         );
 
         Wallet thisWallet = BridgeUtils.getFederationSpendWallet(
-            Context.getOrCreate(networkParameters),
+            Context.getOrCreate(NetworkParameters.fromID(NetworkParameters.ID_REGTEST)),
```

Comment:
shouldnt this be ID_MAINNET? above the bridgeConstants are created with mainnet

Reply:
You are right, corrected